plugin/src/main/java/com/denizenscript/denizen/Denizen.java

```diff
@@ -498,6 +498,9 @@ public void reloadConfig() {
         if (!CoreConfiguration.defaultDebugMode) {
             getLogger().warning("Debug is disabled in the Denizen config. This is almost always a mistake, and should not be done in the majority of cases.");
         }
+        if (Settings.cache_legacySpigotNamesSupport) {
+            getLogger().warning("LEGACY SPIGOT NAMES SUPPORT ENABLED! This should only be used if you have scripts with legacy Spigot names, and should otherwise be disabled; see config.yml for more information.");

```

Comment:
This should be a regular debug log. You copied the special log notice from above which is a very special case of debug being disabled